assets/dev/js/editor/controls/icons.js

```diff
@@ -0,0 +1,212 @@
+const ControlMultipleBaseItemView = require( 'elementor-controls/base-multiple' );
+import IconLibrary from './../components/icons-manager/classes/icon-library';
+
+class ControlIconsView extends ControlMultipleBaseItemView {
+	constructor( ...args ) {
+		super( ...args );
+		this.cache = {
+			loaded: false,
+			dialog: false,
+			enableClicked: false,
+		};
+	}
+
+	enqueueIconFonts( iconType ) {
+		const iconSetting = elementor.helpers.getIconLibrarySettings( iconType );
+		if ( false === iconSetting ) {
+			return;
+		}
+
+		if ( iconSetting.enqueue ) {
+			iconSetting.enqueue.forEach( ( assetURL ) => {
+				elementor.helpers.enqueueStylesheet( assetURL, false );
+			} );
+		}
+
+		if ( iconSetting.url ) {
+			elementor.helpers.enqueueStylesheet( iconSetting.url, false );
+		}
+	}
+
+	ui() {
+		const ui = super.ui();
+		ui.frameOpeners = '.elementor-control-preview-area';
+		ui.svgUploader = '.elementor-control-svg-uploader';
+		ui.iconPicker = '.elementor-control-icon-picker';
+		ui.deleteButton = '.elementor-control-icon-delete';
+		ui.previewContainer = '.elementor-control-icons-preview';
+		ui.previewPlaceholder = '.elementor-control-icons-preview-placeholder';
+
+		return ui;
+	}
+
+	onRender() {
+		super.onRender();
+		if ( ! this.cache.loaded ) {
+			elementor.config.icons.forEach( ( library ) => {
+				if ( 'all' === library.name ) {
+					return;
+				}
+				IconLibrary.initIconType( library );
+			} );
+			this.cache.loaded = true;
+		}
+	}
+
+	events() {
+		return _.extend( ControlMultipleBaseItemView.prototype.events.apply( this, arguments ), {
```

Comment:
Please avoid using of `_` methods, since we should stop using this library. Use jQuery instead. Same for all other instances.